gateway-server/src/test/java/org/apache/knox/gateway/services/factory/ServiceFactoryTest.java

```diff
@@ -52,7 +52,7 @@
 
   protected final GatewayServices gatewayServices = EasyMock.createNiceMock(GatewayServices.class);
   protected final GatewayConfig gatewayConfig = EasyMock.createNiceMock(GatewayConfig.class);
-  protected final Map<String, String> options = Collections.emptyMap();
+  protected final Map<String, String> options = new HashMap<>();
```

Comment:
Any reason why we need a mutable map?

Reply:
We _might_ need. In one of our tests, I modified `options` (removed that since then) and the test failed since the map was immutable.

Reply:
Does the mutability of the map matter for a _test_?

Reply:
Yes. Since this is a base class for other tests this `options` instance is used within test classes.